src/core/core.scale.js

```diff
@@ -634,13 +634,26 @@ module.exports = Element.extend({
 		var cosRotation = Math.cos(labelRotationRadians);
 		var longestRotatedLabel = me.longestLabelWidth * cosRotation;
 		var result = [];
-		var i, tick, shouldSkip;
-
+		var i, tick;
+
+		// Specify the label spacing
+		var labelSpacing;
+		if (optionTicks.labelSpacing) {
+			labelSpacing = optionTicks.labelSpacing;
+			// If not an integer, reject
```

Comment:
Either expand the comment here to explain why `labelSpacing` must be an integer and document the restriction in `linear.md` or remove the restriction. I'm not sure your assumption that the ticks must be integers is true?